release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -1394,6 +1395,50 @@ The `OperatorSource` resource, part of the Marketplace API for the Operator Fram
 
 All MongoDB-based samples have been replaced, deprecated, or removed.
 
+[id="ocp-4-6-images-removed-from-samples-imagestreams"]
+==== Images removed from samples imagestreams
+
+The following images are no longer included in the samples imagestreams provided with {product-title}:
+
+----
+registry.redhat.io/rhscl/mysql-57-rhel7
+registry.redhat.io/rhscl/mongodb-34-rhel7
+registry.redhat.io/rhscl/nginx-114-rhel7
+registry.redhat.io/ubi7/php-72
+registry.redhat.io/rhscl/php-72-rhel7
+registry.redhat.io/rhscl/postgresql-96-rhel7
+registry.redhat.io/rhscl/postgresql-96-rhel7
+registry.redhat.io/ubi7/python-36
+registry.redhat.io/rhscl/python-36-rhel7
+registry.access.redhat.com/ubi7/php-72
+registry.access.redhat.com/ubi7/python-36
+registry.redhat.io/dotnet/dotnet-31-rhel7:3.1
+registry.redhat.io/dotnet/dotnet-21-rhel7:2.1
+registry.redhat.io/dotnet/dotnet-31-runtime-rhel7:3.1
+registry.redhat.io/dotnet/dotnet-21-runtime-rhel7:2.1
+registry.redhat.io/fuse7/fuse-karaf-openshift:1.6
+registry.redhat.io/devtools/go-toolset-rhel7:1.11.5
+registry.redhat.io/rhscl/mariadb-102-rhel7
+registry.redhat.io/rhscl/nginx-114-rhel7
+registry.redhat.io/rhscl/php-72-rhel7
+registry.redhat.io/rhscl/postgresql-96-rhel7
+registry.redhat.io/rhscl/python-36-rhel7
+registry.redhat.io/rhdm-7/rhdm-decisioncentral-rhel8:7.7.0
+registry.redhat.io/rhdm-7/rhdm-decisioncentral-rhel8:7.7.1
+registry.redhat.io/rhdm-7/rhdm-kieserver-rhel8:7.7.0
+registry.redhat.io/rhdm-7/rhdm-kieserver-rhel8:7.7.1
+registry.redhat.io/rhpam-7/rhpam-businesscentral-monitoring-rhel8:7.7.0
+registry.redhat.io/rhpam-7/rhpam-businesscentral-monitoring-rhel8:7.7.1
+registry.redhat.io/rhpam-7/rhpam-businesscentral-rhel8:7.7.0
+registry.redhat.io/rhpam-7/rhpam-businesscentral-rhel8:7.7.1
+registry.redhat.io/rhpam-7/rhpam-kieserver-rhel8:7.7.0
+registry.redhat.io/rhpam-7/rhpam-kieserver-rhel8:7.7.1
+registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8:7.7.0
+registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8:7.7.1
+registry.redhat.io/rhscl/ruby-27-rhel7
```

Comment:
We don't remove imagestream for image `registry.redhat.io/rhscl/ruby-27-rhel7`
https://github.com/openshift/cluster-samples-operator/blob/release-4.6/assets/operator/ocp-x86_64/ruby/imagestreams/ruby-rhel.json#L73

Reply:
updated